src/api/resources/base.py

```diff
@@ -475,14 +466,37 @@ class Meta:
             "site_within",
             "country_id",
             "country_name",
-            "sample_date",
             "tag_id",
             "tag_name",
             "reef_type",
             "reef_zone",
             "reef_exposure",
             "management_id",
             "management_name",
+        ]
+
+    def full_management_name(self, queryset, name, value):
+        fields = ["management_name", "management_name_secondary"]
+        return self.str_or_lookup(queryset, fields, value, lookup_expr="icontains")
+
+
+class BaseSEFilterSet(AggregatedViewFilterSet):
+    id = BaseInFilter(method="id_lookup")
+    sample_date = DateFromToRangeFilter()
+    management_est_year = DateFromToRangeFilter()
```

Comment:
does this work for an integer field? (https://django-filter.readthedocs.io/en/stable/ref/filters.html?highlight=DateFromToRangeFilter#datefromtorangefilter)